package.json

```diff
@@ -11,12 +11,13 @@
   },
   "main": "./lib/index.js",
   "scripts": {
-    "lint": "npm install && eslint lib/**/*.js test/**/*.js",
-    "test": "npm install && mocha test/**/*.js"
+    "lint": "eslint --fix lib/**/*.js test/**/*.js",
```

Comment:
Would rather not impose editor config here.

Reply:
So, remove the `--fix` flag?

Reply:
Yup, that works.